build/lib/src/generate/run_builder.dart

```diff
@@ -14,23 +14,21 @@ import '../builder/builder.dart';
 import '../builder/logging.dart';
 import 'expected_outputs.dart';
 
-/// Run [builder] on each asset in [inputs].
+/// Run [builder] on with asset in [inputs] as the primary input.
```

Comment:
This reads odd. Were you trying to say 'Run [builder] with each asset in [inputs] as the primary input.'